plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -220,30 +225,23 @@ class ServiceActionDisabledModal extends React.Component {
 
   getServiceEditMessage() {
     const { intl } = this.props;
-    const command = this.getUpdateCommand();
 
     return (
-      <div>
-        <p>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1"
-          })}
+      <MountService.Mount
+        type={"ServiceEditMessage:Modal"}
+        getClipboardTrigger={this.getClipboardTrigger}
```

Comment:
I don't understand what would be a better alternative. `getUpdateCommand` has other consumers in this file, and relies on `isSDKService`. Isn't the alternative to duplicate this into private-plugins?